tools/cli/help.txt

```diff
@@ -532,7 +532,65 @@ Options:
   --verbose         Print all output from builds logs.
   --no-lint         Don't run linters used by the tested packages on every
                     test app rebuild.
+>>> test-app
+Test the application
+Usage: meteor test-app [options]
+       meteor test-app --unit [options]
+       meteor test-app --integration [options]
+       
+Runs tests against the application, either in *unit* or *integration* 
+test mode. meteor test-app will start a special app based on a test driver
+which handles the task of running tests and displaying the results in the 
+browser when you visit it.
+
+In unit test mode, no files in your application are eagerly loaded, aside
+from test files (in you tests/ directory or named *.tests.*). You can
```

Comment:
FWIW, Right now it also finds files named `tests.*` or `test.*` or `*.test.*`. (We can change that if we want)